__tests__/Unit/Components/Tasks/TaskDetails.test.tsx

```diff
@@ -191,228 +190,211 @@ describe('TaskDetails Page', () => {
+test('should call onSave and reset state when clicked', async () => {
+    renderWithRouter(
+        <Provider store={store()}>
+            <TaskDetails taskID={details.taskID} />
+        </Provider>,
+        {}
+    );
 
-        renderWithRouter(
-            <Provider store={store()}>
-                <TaskDetails taskID={details.taskID} />
-                <ToastContainer />
-            </Provider>,
-            {}
-        );
-        await waitFor(() => {
-            const editButton = screen.getByRole('button', { name: 'Edit' });
-            fireEvent.click(editButton);
-        });
-        const textareaElement = screen.getByTestId('title-textarea');
-        fireEvent.change(textareaElement, {
-            target: { name: 'title', value: 'New Title' },
-        });
-        await waitFor(async () => {
-            const saveButton = screen.getByRole('button', { name: 'Save' });
-            fireEvent.click(saveButton);
-            expect(
-                await screen.findByText(/Successfully saved/i)
-            ).not.toBeNull();
-        });
-    });
-    test('should not update the title and description with the same values', async () => {
-        server.use(...taskDetailsHandler);
-        renderWithRouter(
-            <Provider store={store()}>
-                <TaskDetails taskID={details.taskID} />
-                <ToastContainer />
-            </Provider>,
-            {}
-        );
-        const editButton = await screen.findByRole('button', { name: 'Edit' });
+    await waitFor(() => {
+        const editButton = screen.getByRole('button', { name: 'Edit' });
         fireEvent.click(editButton);
-        const textareaElement = await screen.findByTestId('title-textarea');
-        fireEvent.change(textareaElement, {
-            target: { name: 'title', value: 'test 1 for drag and drop' },
-        });
-        await waitFor(async () => {
-            const saveButton = await screen.findByRole('button', {
-                name: 'Save',
-            });
-            fireEvent.click(saveButton);
-        });
-        expect(screen.queryByText(/Successfully saved/i)).toBeNull();
     });
 
-    test('Should render No task progress', async () => {
-        renderWithRouter(
-            <Provider store={store()}>
-                <TaskDetails taskID={details.taskID} />
-            </Provider>
-        );
-        let progressUpdatesSection;
-        await waitFor(() => {
-            progressUpdatesSection = screen.getByText('Progress Updates');
-        });
-        expect(progressUpdatesSection).toBeInTheDocument();
-        const noProgressText = screen.getByText('No Progress found');
-        expect(noProgressText).toBeInTheDocument();
+    await waitFor(() => {
+        const saveButton = screen.getByRole('button', { name: 'Save' });
+        fireEvent.click(saveButton);
+        const editButton = screen.getByRole('button', { name: 'Edit' });
+        expect(editButton).toBeInTheDocument();
```

Comment:
https://github.com/Real-Dev-Squad/website-status/pull/1145/files#r1501798000

Reply:
Didn't understand what are you trying to say here.